packages/lib/src/components/Card/Card.tsx

```diff
@@ -57,7 +57,7 @@ export class CardElement extends UIElement<CardConfiguration> {
         _disableClickToPay: false,
         doBinLookup: true,
         // Merge most of CardInput's defaultProps
-        ...reject(['type', 'setComponentRef']).from(CardInputDefaultProps)
+        ...reject(['type', 'setComponentRef', 'onEnterKeyPressed']).from(CardInputDefaultProps)
```

Comment:
It does not make sense to have 'onEnterKeyPress' as default props, since it is not actually a default prop and it shouldn't be. Can't we remove it from there?

Reply:
Remove it from where? `CardInputDefaultProps`?

Reply:
yes

Reply:
Done